pkg/vpa/vpa.go

```diff
@@ -457,3 +484,11 @@ func vpaMinReplicasForResource(obj runtime.Object) (*int32, bool) {
 
 	return &minReplicasInt32, explicit
 }
+
+func vpaRecommenderNameForNamespace(ns *corev1.Namespace) (string, error) {
+	recommenderName, ok := ns.Labels["goldilocks.fairwinds.com/vpa-recommenders"]
```

Comment:
I believe we currently support both labels and annotations for the other settings that affect the VPA configuration. Would it be more appropriate to support both here?

Reply:
I think that makes sense, I can update